src/Service/ComposerService.php

```diff
@@ -454,7 +451,7 @@ public function getSingle(string $package, array $options = []): array
                 // Let's see if we have newer versions than currently installer
                 foreach ($plugin['versions']  as $version => $versionDetails) {
                     if (version_compare($plugin['version'], $version) < 0) {
-                        if (!$plugin['update']) {
+                        if (isset($plugin['update']) === false || empty($plugin['update']) === true) {
```

Comment:
https://www.geeksforgeeks.org/difference-between-isset-and-empty-functions/
![image](https://github.com/CommonGateway/CoreBundle/assets/3785488/b29ce3fb-2f71-440c-89f2-c2487cc79b21)
![image](https://github.com/CommonGateway/CoreBundle/assets/3785488/b3a9e3d5-5c34-45d7-8a1a-544ceda4c361)

Ik denk dat de enige situatie waarin je beide zou willen gebruiken
isset() === true && empty() === false

Reply:
With both checks I think $string = ''; does get through while when using only isset() === false it might not go through because isset accepts empty string as set? Not sure tho.

Reply:
Okay, dove a bit deeper into this, indeed an empty string will be recognised as set, but if in the old version we were able to do !$plugin['update'] we wouldn't need to check if it is set, so we only would need empty($plugin['update']) === true in the check.